hexkit/providers/akafka/testutils.py

```diff
@@ -408,7 +409,8 @@ async def expect_events(
 
 
 async def kafka_fixture_function() -> AsyncGenerator[KafkaFixture, None]:
-    """Pytest fixture for tests depending on the Kafka-base providers."""
+    """Pytest fixture for tests depending on the Kafka-base providers.
+    **Do not call directly** Instead, use get_kafka_fixture()"""
```

Comment:
Just as a general remark, let's follow [PEP 257](https://peps.python.org/pep-0257/) in docstrings.

"one-line summary, followed by a blank line, followed by a more elaborate description"

The point is that some auto doc creation tools split the docstring in summary and description, which are then displayed differently. The summary should fit into one line.

Reply:
I had not seen that PEP yet, thanks for bringing it to my attention. I read through it and reformatted the doc strings I touched in this PR.